src/patterns/molecules/sticky-promo/sticky-promo.hbs

```diff
@@ -2,6 +2,8 @@
 name: Sticky Promo Card
 description: The primary purpose of the Sticky Card component is to ensure that the content it’s promoting remains a constant on the page -- always visible within the viewport no matter a user’s scroll position. Sticky Promo Cards can be used to promote a product and provide a CTA. While the card itself is both content and context agnostic, it should only be used strategically on pages where it will be most effective.
 notes: |
+  - The element is hidden by default. To show it, add `mzp-a-slide-in` to the class list.
```

Comment:
This instruction is confusing, and makes it sound like I should add that class to the HTML by hand. Really the way to display the promo is by firing `Mzp.StickyPromo.open()` with JS, and that function in turn adds the animation class.

So let's change this to something like "The element is hidden by default and made visible by the `Mzp.StickyPromo.open()` function."